client/src/app/site/motions/modules/motion-block/components/motion-block-detail/motion-block-detail.component.html

```diff
@@ -39,9 +39,15 @@ <h2 *ngIf="block">
     (dataSourceChange)="onDataSourceChange($event)"
 >
     <!-- Title column -->
-    <div *pblNgridCellDef="'title'; row as motion; rowContext as rowContext" class="cell-slot fill motion-block-title">
+    <div *pblNgridCellDef="'title'; row as motion; rowContext as rowContext" class="cell-slot fill motion-block-title one-line">
         <a class="detail-link" [routerLink]="motion.getDetailStateURL()" *ngIf="!isMultiSelect"></a>
-        <span>{{ motion.getTitle() }}</span>
+        <div class="ellipsis-overflow">
+            <span *ngIf="motion.identifier">
```

Comment:
Should probably be dependent on the option: `Show the sequential number for a motion`

Reply:
I added the same multi-line layout as in motion-list - with submitters, sqeuential number etc.